introduction/faq.md

```diff
@@ -248,7 +258,7 @@ At the same time, due to the smart use of Xen shared memory, our GUI implementat
 
 ### Why passwordless sudo?
 
-Please refer to [this page](/doc/vm-sudo/).
+We have found that there's really no point in user/root isolation, because all the user data (and VM management interface) is already accessible from dom0 user level; so, there is nothing more to get from a dom0 root account. Read more about: [Passwordless root access in Qubes](https://www.qubes-os.org/doc/vm-sudo/)
```

Comment:
@ninavizz  `/doc/vm-sudo/`  instead of direct link `https://www.qubes-os.org/doc/vm-sudo/`